dev-haskell/editor-open/editor-open-0.6.0.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.1.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="Open the user's $VISUAL or $EDITOR for text input"
```

Comment:
`$VISUAL` and `$EDITOR` will need to be escaped. Worth fixing in hackport as well if it does not do it already.